dbld/build.manifest

```diff
@@ -43,3 +43,4 @@ ubuntu-mantic		python3
 
 fedora			python3
 almalinux		python3
+centos          python3
```

Comment:
match the indentation, also it looks like that in this file indentation is done via tabs and not psaces

Reply:
Seems like my IDE had swapped the tabs for spaces... Fixed